web-ui/src/main/resources/catalog/components/userfeedback/GnUserfeedbackDirective.js

```diff
@@ -0,0 +1,411 @@
+                  return;
+                }
+
+                $http({
+                  method: 'GET',
+                  url: '../api/userfeedback?uuid=' + scope.metatdataUUID + '&maxnumber=3',
+                  isArray: true
+                }).then(function mySuccess(response) {
+                  scope.fewCommentsList = [];
+                  scope.fewCommentsList = scope.fewCommentsList.concat(response.data);
+                  scope.showButtonAllComments = true;
+                  scope.showModal = false;
+                }, function myError(response) {
+                  console.log('gnUserfeedback.loadComments ' + scope.metatdataUUID);
+                  console.log(response.statusText);
+                });
+
+
+                $http({
+                  method: 'GET',
+                  url: '../api/records/' + scope.metatdataUUID + '/userfeedbackrating',
+                  isArray: false
+                }).then(function mySuccess(response) {
+                  scope.rating = null;
+                  scope.rating = response.data;
+                }, function myError(response) {
+                  console.log('gnUserfeedback.loadComments ' + scope.metatdataUUID);
+                  console.log(response.statusText);
+                });
+              };
+            }
+
+          };
+        }]);
+
+
+  module.directive(
+      'gnUserfeedbackfull', ['$http', 'gnSearchLocation', '$window', '$translate', 'gnConfig',
+        function($http, gnSearchLocation, $window, $translate, gnConfig) {
+          return {
+            restrict: 'AEC',
+            replace: true,
+            controller: 'gnUserfeedbackControllerFull',
+            scope: {
+              parentUuid: '@gnUserfeedbackfull',
+              userName: '@gnUser'
+            },
+            templateUrl: '../../catalog/components/userfeedback/partials/userfeedbackfull.html',
+            link: function(scope) {
+
+
+
+              scope.$watch('parentUuid', function(newValue, oldValue) {
+
+                scope.metatdataUUID = newValue;
+
+              });
+
+              scope.$watch('userName', function(newValue, oldValue) {
+
+                if (newValue) {
+                  scope.loggedIn = true;
+                  scope.authorNameValue = newValue;
+                } else {
+                  scope.loggedIn = false;
+                }
+
+              });
+
+
+
+              scope.initPopup = function() {
+                scope.fullCommentsList = [];
+                scope.rating = null;
+
+                if (angular.isUndefined(scope.metatdataUUID) || scope.metatdataUUID == null || scope.metatdataUUID == '') {
+                  console.log('Metadata UUID is null');
+                  return;
+                }
+
+                $http({
+                  method: 'GET',
+                  url: '../api/userfeedback?full=true&uuid=' + scope.metatdataUUID,
```

Comment:
I don't see this full parameter in the server side. What is it for ?

Reply:
No, it's not used anymore